project/Dependencies.scala

```diff
@@ -120,14 +125,14 @@ object Dependencies {
   private val circeYamlDependency = "io.circe" %% "circe-yaml" % circeYamlV
 
   private val circeDependencies = List(
-    "core",
-    "parser",
-    "generic",
-    "generic-extras",
-    "shapes",
-    "refined",
-    "literal"
-  ).map(m => "io.circe" %% s"circe-$m" % circeV) :+ circeYamlDependency
+    "io.circe" %% s"circe-core" % circeV,
+    "io.circe" %% s"circe-parser" % circeV,
+    "io.circe" %% s"circe-generic" % circeV,
+    "io.circe" %% s"circe-generic-extras" % circeV,
+    "io.circe" %% s"circe-shapes" % circeV,
+    "io.circe" %% s"circe-refined" % circeV,
+    "io.circe" %% s"circe-literal" % circeV
+  ) :+ circeYamlDependency
```

Comment:
ooc was this de-DRYed because it confused Steward?

Reply:
Judging by the titles I believe the intent of Scala Steward PRs #5640 and #5641 was to:

1. ) Upgrade all of these dependencies except circe-generic-extras to 0.12.3
2. ) Upgrade circe-generic-extras to 0.12.2

Those aren't the code changes the Scala Steward PRs actually produce (all of these dependencies flip from 0.12.1 to 0.12.3 to 0.12.2), but that's another matter. 🙂 Here's [what I did](https://github.com/broadinstitute/cromwell/pull/5711/commits/7f28d1a9741b30f12811366648082f92a5634937) in my bite-size #5711 PR.

Reply:
Yeah, our ad-hoc dependency name generation seemed to be causing scala-steward some issues with version tracking and concurrent-updates-to-different-versions, so I tried to make it easier for it.